examples/mount/Kconfig

```diff
@@ -35,7 +35,7 @@ if !EXAMPLES_MOUNT_BLOCKDEVICE
 
 config EXAMPLES_MOUNT_NSECTORS
 	int "RAM disk number of sectors"
-	default DEFAULT_TASK_STACKSIZE
+	default 2048
```

Comment:
Isn't that supposed to be the opposite way?

Reply:
Never mind, I see it has nothing to do with a task.

Reply:
This is a wrong change by:
commit e7156be066c7427e2afb56521c3eaf56d7623e25
Author: YAMAMOTO Takashi <yamamoto@midokura.com>
Date:   Fri Mar 27 14:10:33 2020 +0900

    Change the defaults of stack size configs to DEFAULT_TASK_STACKSIZE
    
    This commit changes only ones with the default 2048 and
    leaves the others.
    E.g. this leaves SYSTEM_RAMTEST_STACKSIZE, whose default is 1024.
    I guess those need to be inspected one-by-one.